### ISSUE TITLE
Multiple events per day

### ISSUE THREAD

**Issue by mrcasals** [christopherdro/react-native-calendar#83]
Is there any way to have show multiple events per day, right now?

I've tried adding the same day multiple times in the `event` array with different styles, but only the last event is shown:

```
  <Calendar
      weekStart={1}
      scrollEnabled
      showControls
      showEventIndicators
      eventDates={['2016-12-03', '2016-12-04', '2016-12-28', '2016-12-30']}
      events={[{date: '2016-12-04', eventIndicator: {backgroundColor: 'powderblue'}}, {date: '2016-12-04', eventIndicator: {backgroundColor: 'red'}}]}
  />
```

Should appear both blue and red indicators on 2016-12-04:
![](https://i.imgur.com/Angmc2y.png)

**Comment by hejiaji** [comment]
U mean render multiple dots in one day? why would you need that function? the dot means that you have event this very day..  so if you wanna checkout what's going on in that day, you should click it and jump to another page to show up how many activities and details in that day.

@mrcasals so we don't support display different dots.

**Comment by christopherdro** [comment]
I agree with @hejiaji. Single dot to indicate the day has a minimum of 1 event.

**Comment by FasterHQ** [comment]
Yeah, but there can be different types of events for a day. Like Google does. It would make more sense to be expandable like that. Would you guys accept a merge request if I added this feature?

**Comment by christopherdro** [comment]
Sure. I'll be more than happy to review once a PR is up!

**Comment by n1sivakumar** [comment]
Hello. I also would like to have this feature. Trying to use this component to incorporate multiple calendars (Personal, Work, Group, ...) for the same account. So, a different color coded event indicator would be very useful.
